poll/public/js/poll_edit.js

```diff
@@ -25,7 +25,9 @@ function PollEditUtil(runtime, element, pollType) {
                             // The degree of precision on date should be precise enough to avoid
                             // collisions in the real world.
                             var bottom = $(button_mapping[context_key]['bottomMarker']);
-                            var new_item = $(self.answerTemplate(button_mapping[context_key]['itemList']));
+                            var new_item_dict = self.extend({}, button_mapping[context_key]['template']);
+                            new_item_dict['key'] = Date.now();
+                            var new_item = $(self.answerTemplate({'items': [new_item_dict]}));
```

Comment:
@Kelketek Thanks for the explanation, I see. Can you add a comment to explain this? To make it easier to understand the constraint here when other people work on it later on (and for future you too once you'll have forgot the details : ) ). 

Also, for ids it would be better to use unique ids (like https://github.com/pnegri/uuid-js ) - it would guarantee the uniqueness, and make it more obvious that this is an id field, not a date field.